### DISCUSSION TITLE
Packaging

### DISCUSSION THREAD

**Discussion by textbrowser** [textbrowser/biblioteq#346] [General]
The following new package will be introduced in 2024: MacOS Silicon.
The following packages will be deprecated in 2024: Intel MacOS, PiOS ARM32. Packages including corrections are exempt.

**Comment by meteos77** [comment]
Thanks for all the package versions! PIOS64, Win :-)

**Comment by meteos77** [comment]
I have a volunteer who has a Mac but can't launch BiblioteQ (I don't have the details yet: Mac end of 2019).

What MacOS configuration do I need for BiblioteQ to be OK?

For package version 2023-12-25  : Mac Intel but MacOS version ? do you recommend ?

**Comment by textbrowser** [comment]
The latest release for MacOS is Intel. If it doesn't work, it doesn't work. I think the target is a minimum of 11. Of course, creating from source. And again, packaging is hard.

**Comment by textbrowser** [comment]
Have you considered a Mac Mini for the library? Get together some funds from the folks. It's like Linux but a bit nicer looking. And the Minis are not too expensive. Only Silicon and BQ is fine on them. There is also the low power consumption.

**Reply by meteos77** [reply]
I don't think changing PCs will be on the agenda before ... 10 years :-)

PC portable ASUS P550L - Model X550LA / Processeur : Intel i34030U / Mem : 4Go
	- Disque Dur : SSD Samsung 870EVO Sata 2.5"
	- Écran : 19 pouces 16:9 - Résolution 1366x768.
	- Port USB : 3 (2xUSB3.0 et 1xUSB2)
	- OS : LinuxMint 21.1 (2023-05)
	
We are a small association with no money ;  budget: 7€ per family per year maximum 30 families
laptop donated by the town hall ... recovery

But BiblioteQ works admirably well with 4600 books without covers with entries per page: unlimited.

Having a mac mini (M2?) is still expensive here?  700€ = between 30 - 35 books.

**Comment by meteos77** [comment]
>  It's like Linux but a bit nicer looking

I saw that it was possible to run macOS in virtualbox ... as soon as I have a moment I'll give it a try.

**Comment by textbrowser** [comment]
Open a Fund-Me petition. Or, a local box for donations towards newer hardware. Can you petition the town? "Hey, we need money for this library." Younger generations are finally discovering the library, like, they didn't imagine libraries existed. I blame the parents. Anyway, Mac Mini is small and conserves power. MacOS includes accessibility features. Perhaps better than Linux and Windows.

**Comment by charabaruk** [comment]
I'd like to announce that as of May 2025, BiblioteQ is available for Arch users in the AUR.

**Comment by textbrowser** [comment]
That good.
